RbootHttpUpdater - Compiler Syntax error

I suspect I have a more recent compiler than the one you use.

C+ /opt/Sming/Sming/Core/Network/RbootHttpUpdater.cpp
/opt/Sming/Sming/Core/Network/RbootHttpUpdater.cpp: In member function ‘bool RbootHttpUpdater::addItem(const String&, RbootOutputStream*)’:
/opt/Sming/Sming/Core/Network/RbootHttpUpdater.cpp:39:46: error: ‘void* memset(void*, int, size_t)’ **clearing an object of type ‘struct RbootHttpUpdaterItem’ with no trivial copy-assignment; use assignment or value-initialization** instead [-Werror=class-memaccess]
  memset(&add, 0, sizeof(RbootHttpUpdaterItem));

> /opt/Sming/Sming/Core/Network/RbootHttpUpdater.cpp:39:46: error: ‘void* memset(void*, int, size_t)’ 

Thanks for pointing this out. I will remove that line as it is not really needed. PR is on the way.

--- a/Sming/Core/Network/RbootHttpUpdater.cpp
+++ b/Sming/Core/Network/RbootHttpUpdater.cpp
@@ -36,7 +36,6 @@ bool RbootHttpUpdater::addItem(const String& firmwareFileUrl, RbootOutputStream*
 	}
 
 	RbootHttpUpdaterItem add;
-	memset(&add, 0, sizeof(RbootHttpUpdaterItem));
 	add.targetOffset = stream->getStartAddress();
 	add.url = firmwareFileUrl;
 	add.size = stream->getMaxLength();

